src/main/pages/che-7/installation-guide/assembly_airgap-installation.adoc

```diff
@@ -0,0 +1,161 @@
+---
+title: Installing Che in air-gapped or offline mode
+keywords: air-gap, airgap, offline
+tags: [installation]
+sidebar: che_7_docs
+permalink: che-7/airgap-install/
+folder: che-7/installation-guide
+summary:
+---
+
+:parent-context-of-installing-che-in-air-gapped-or-offline-mode: {context}
+
+[id='installing-che-in-air-gapped-or-offline-mode_{context}']
+= Installing Che in air-gapped or offline mode
+
+:context: installing-che-in-air-gapped-or-offline-mode
+
+## Introduction
+By default, Eclipse Che's workspaces reference various external resources, whether they are docker images available on public registries, or sample projects hosted on GitHub.
+
+In some situations, it may be desirable to deploy Che in an environment where these external resources are not available (e.g. on a cluster that is not exposed to the public internet). Doing this requires a few customizations:
+
+* Certain images need to be built containing external resources (workspace plugins, projects)
+* All images to be used for running Che need to be pulled to a registry that is visible
+* Che needs to be configured to use these images
+
+The following sections describe the steps necessary to deploy Che in an airgapped Kubernetes or OpenShift cluster.
+
+[#offline-registries]
+## Building offline versions of the plugin and devfile registry
+[NOTE]
+====
+This section describes how to build _offline_ registry images. For additional information, see the main link:{site-baseurl}che-7/building-and-running-a-custom-registry-image/[building and running a custom registry image] doc.
+====
+
+The main purpose of building offline regsitry images is to ensure that workspaces can be started without relying on resources from the outside internet. For the plugin registry, this means building a registry that serves all plugin or extension artifacts in additiona to plugin metadata. Similarly, an offline devfile registry serves all sample projects referenced in devfiles as `zip` files.
```

Comment:
There is a typo, "in additiona to" -> "in addition to"?

Reply:
Good catch, fixed.